README.md

```diff
@@ -225,9 +228,175 @@ If the modification of the Application resource is completed successfully, the `
 
 Extensions are optional additions to the core specification as defined above.  Implementation and support of these specifications are not required in order for a platform to be considered compliant.  However, if the features addressed by these specifications are supported a platform **MUST** be in compliance with the specification that governs that feature.
 
+## Binding `Secret` Generation Strategies
```

Comment:
Things have loosened up in the core spec on this a bit and `type` (formerly `kind`, changed by request) is now a SHOULD for everything but the application projection.  This feels right as well, as the requirement _isn't_ that a the secret, wherever it is must have `type`, just that an application can depend on `type` being there.   Therefore, it can come from the original provisioned service, it can be added in a synthesized provisioned service, a mapping, or even the [magic key](https://github.com/application-stacks/service-binding-specification/pull/48/files#diff-04c6e90faac2675aa89e2176d2eec7d8R185) in the service binding.  The latter feels like the most common strategy at first, to slowly be replaced by services adding it to their secrets natively.

Reply:
The loosening definitely makes this less of a priority, however it would still be good to include an example of defining a property sourced from a literal value, rather than a mapping from existing data.

Reply:
@arthurdm How would a descriptor/annotation work here?

Reply:
That's a good question.  We don't have any examples of just pointing to a literal.  I would image an annotation would look like this for example: `service.binding/type: <literal>`

I am not sure it would make sense to have `x-descriptor` pointing to a literal, because `x-descriptors` are always attached to an existing item from the CSV.  I would expect that `status.type` be something exposed by the bindable service.  

@sbose78 - have you had talks in SBO about literals from the annotation?